json/nl.json

```diff
@@ -13,7 +13,6 @@
     "Error": "Fout",
     "Forbidden": "Geen toegang",
     "Forgot your password? No problem. Just let us know your email address and we will email you a password reset link that will allow you to choose a new one.": "Forgot your password? No problem. Just let us know your email address and we will email you a password reset link that will allow you to choose a new one.",
-    "Forgot your password?": "Forgot your password?",
```

Comment:
@Lubje You should **NOT delete** this key because it is a current [translation](https://github.com/laravel/jetstream/blob/1.x/stubs/resources/views/auth/login.blade.php#L38) in laravel Jetstream.

Remember that the JSON keys are *case sensitive* therefore `'Forgot your password?'` is different to `'Forgot Your Password?'`.

Therefore I consider that neither of the two keys should be deleted or modified because both are used:
* `'Forgot your password?'` in [Laravel Jetstream](https://github.com/laravel/jetstream/blob/1.x/stubs/resources/views/auth/login.blade.php#L38) and 
* `'Forgot Your Password?'` in [Laravel UI](https://github.com/laravel/ui/blob/3.x/src/Auth/bootstrap-stubs/auth/passwords/confirm.stub#L38)

If we delete the second we would lose the translation for those who still use Laravel UI

I suggest translate this key instead of delete